goes-plume-viewer/src/components/drawer/index.jsx

```diff
@@ -122,17 +137,25 @@ export function PersistentDrawerRight({open, setOpen, selectedPlume, collectionI
             </Typography>
           </HorizontalLayout>
         </DrawerHeader>
-          { selectedPlumeMeta && selectedPlume && <PlumeCard
-              plumeSourceName={selectedPlumeMeta.plumeSourceName}
-              imageUrl={`${process.env.REACT_APP_RASTER_API_URL}/collections/${collectionId}/items/${selectedPlume["data"]["id"]}/preview.png?assets=rad&rescale=${VMIN}%2C${VMAX}&colormap_name=${colorMap}`}
-              tiffUrl={`${process.env.REACT_APP_RASTER_API_URL}/collections/${collectionId}/items/${selectedPlume["data"]["id"]}/preview.png?assets=rad&rescale=${VMIN}%2C${VMAX}&colormap_name=${colorMap}`}
-              lon={selectedPlumeMeta.lon}
-              lat={selectedPlumeMeta.lat}
-              totalReleaseMass={selectedPlumeMeta.totalReleaseMass}
-              colEnhancements={selectedPlumeMeta.colEnhancements}
-              startDatetime={selectedPlumeMeta.startDatetime}
-              endDatetime={selectedPlumeMeta.endDatetime}
-            />
+          { !!selectedPlumeMetas.length &&
+            selectedPlumeMetas.map(selectedPlumeMeta => (
+              <PlumeCard
+                key={selectedPlumeMeta.id}
+                plumeSourceId={selectedPlumeMeta.id}
+                plumeSourceName={selectedPlumeMeta.id.replace(/_/g, " ")}
+                imageUrl={`${process.env.REACT_APP_RASTER_API_URL}/collections/${collectionId}/items/${plumesMap[selectedPlumeMeta.id].representationalPlume.id}/preview.png?assets=rad&rescale=${VMIN}%2C${VMAX}&colormap_name=${colorMap}`}
+                tiffUrl={`${process.env.REACT_APP_RASTER_API_URL}/collections/${collectionId}/items/${plumesMap[selectedPlumeMeta.id].representationalPlume.id}/preview.png?assets=rad&rescale=${VMIN}%2C${VMAX}&colormap_name=${colorMap}`}
```

Comment:
why two of the same thing? 🤔

Reply:
will change that once i get the tillUrl. It will be something different.

Reply:
will be working on it in a different branch.